diff --git a/TSB_AD/model_wrapper.py b/TSB_AD/model_wrapper.py
index 94e4c91..86ffd62 100644
--- a/TSB_AD/model_wrapper.py
+++ b/TSB_AD/model_wrapper.py
@@ -278,6 +278,13 @@ def run_AutoEncoder(data_train, data_test, window_size=100, hidden_neurons=[64,
     score = clf.decision_function(data_test)
     return score.ravel()
 
+def run_AutoEncoder_2(data_train, data_test, window_size=100, hidden_neurons=[64, 32], n_jobs=1):
+    from .models.AE_2 import AutoEncoder
+    clf = AutoEncoder(slidingWindow=window_size, hidden_neurons=hidden_neurons, batch_size=128, epochs=50)
+    clf.fit(data_train)
+    score = clf.decision_function(data_test)
+    return score.ravel()
+
 def run_CNN(data_train, data_test, window_size=100, num_channel=[32, 32, 40], lr=0.0008, n_jobs=1):
     from .models.CNN import CNN
     clf = CNN(window_size=window_size, num_channel=num_channel, feats=data_test.shape[1], lr=lr, batch_size=128)
diff --git a/TSB_AD/models/KMeansAD.py b/TSB_AD/models/KMeansAD.py
index 22783ad..3dcb4d0 100644
--- a/TSB_AD/models/KMeansAD.py
+++ b/TSB_AD/models/KMeansAD.py
@@ -22,13 +22,13 @@ class KMeansAD(BaseEstimator, OutlierMixin):
         flat_shape = (X.shape[0] - (self.window_size - 1), -1)  # in case we have a multivariate TS
         slides = sliding_window_view(X, window_shape=self.window_size, axis=0).reshape(flat_shape)[::self.stride, :]
         self.padding_length = X.shape[0] - (slides.shape[0] * self.stride + self.window_size - self.stride)
-        print(f"Required padding_length={self.padding_length}")
+        # print(f"Required padding_length={self.padding_length}")
         if self.normalize: slides = zscore(slides, axis=1, ddof=1)
         return slides
 
     def _custom_reverse_windowing(self, scores: np.ndarray) -> np.ndarray:
-        print("Reversing window-based scores to point-based scores:")
-        print(f"Before reverse-windowing: scores.shape={scores.shape}")
+        # print("Reversing window-based scores to point-based scores:")
+        # print(f"Before reverse-windowing: scores.shape={scores.shape}")
         # compute begin and end indices of windows
         begins = np.array([i * self.stride for i in range(scores.shape[0])])
         ends = begins + self.window_size
@@ -46,7 +46,7 @@ class KMeansAD(BaseEstimator, OutlierMixin):
 
         # replace untouched indices with 0 (especially for the padding at the end)
         np.nan_to_num(mapped, copy=False)
-        print(f"After reverse-windowing: scores.shape={mapped.shape}")
+        # print(f"After reverse-windowing: scores.shape={mapped.shape}")
         return mapped
 
     def fit(self, X: np.ndarray, y=None, preprocess=True) -> 'KMeansAD':
